maxim/logger/openai/utils.py

```
@@ -14,52 +14,54 @@ def parse_message_param(
             if isinstance(content, list):
                 # Handle content blocks for multimodal
                 text_content = ""
                 for block in content:
                     if isinstance(block, dict) and block.get("type") == "text":
                         text_content += block.get("text", "")
                 parsed_messages.append(
-                    GenerationRequestMessage(
-                        role=role,
-                        content=text_content
-                    )
+                    GenerationRequestMessage(role=role, content=text_content)
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)